addons/project_todo/tests/test_mail_activity_todo_create.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.project.tests.test_project_base import TestProjectCommon
+import datetime
+from markupsafe import Markup
+
+class TestMailActivityTodo(TestProjectCommon):
+    @classmethod
```

Comment:
Another style comment: surround methods with one blank line
For reference: https://peps.python.org/pep-0008/#blank-lines

Reply:
@MissingNoShiny Oh, I did not know that, each times, I asked to remove it when it is the first method in the class as it is the case here but okay, let's do that to be coherent in our reviews. 😁